kasa/smart/modules/module.py

```diff
@@ -0,0 +1,6 @@
+"""Base class for all module implementations."""
+from ...iot.modules.module import Module as IotModule
```

Comment:
We shouldn't depend on anything from the iot package in smart, and other way around.

Reply:
As per https://github.com/python-kasa/python-kasa/pull/716#discussion_r1470741168 this is temporary

Reply:
Now resolved